apps/dispatch/src/app/services/dispatch/dispatch.service.ts

```diff
@@ -355,6 +356,15 @@ export class DispatchService {
     );
   }
 
+  public updateSimulationRun(
+    uuid: string,
+    update: UpdateSimulationRun,
+  ) {
+    return this.http.patch(`${urls.dispatchApi}runs/${uuid}`, update, {
```

Comment:
The dispatch app cannot use this method as it requires authentication. A token that allows this method to be called would allow any simulation to be updated and the token would need to be embedded in the front end. We can do something like this only  if we require user accounts to publish models

Reply:
This was fine to merge. 

This is currently disabled. This is just a skeleton. This needs to be replaced with the final method to publish simulation runs.